### DISCUSSION TITLE
Is there an allow list for a running docker instance?

### DISCUSSION THREAD

**Discussion by Bschim** [microsoft/presidio#1373] [Q&A]
Hi!

I am trying to get presidio work with LiteLLM and I was wondering if I could create an allow_list when running presidio in a docker container?

Thanks!

**Comment by ghost** [comment]
Hey @Bschim - litellm maintainer here - is this what you're referring to - https://microsoft.github.io/presidio/api/analyzer_python/#presidio_analyzer.AnalyzerEngine.analyze

I think we can send it as part of the api call? 

P.S.: Would love to learn how you're using litellm! Attaching my calendly, if that helps - https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat

**Comment by omri374** [comment]
Hi @Bschim, we didn't get a chance to add this particular field to the API yet. If you'd like, it requires a relatively simple change to the `AnalyzerRequest` class: https://github.com/microsoft/presidio/blob/main/presidio-analyzer/presidio_analyzer/analyzer_request.py
and an update in `app.py`: https://github.com/microsoft/presidio/blob/2805c86221498f6e6843e608246af136b3137158/presidio-analyzer/app.py#L68

If you're interested (or anyone else) to do a PR adding this, that'd be amazing.

**Comment by Bschim** [comment]
Thanks for the tips guys @krrishdholakia @omri374! I would like to add the field into the API, but I am uncertain if I have the time to do so. It is currently not a priority within my usecase.
